operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -130,6 +130,40 @@ data:
       - cluster:policy_governance_info:propagated_noncompliant_count
       - policy:policy_governance_info:propagated_count
       - policy:policy_governance_info:propagated_noncompliant_count
+      ### metrics for ocp 311 start
+      - node:node_cpu_utilisation:avg1m
+      - kube_node_labels
+      - 'node_namespace_pod:kube_pod_info:'
+      - container_memory_usage_bytes
+      - node_memory_MemTotal_bytes
+      #- node_memory_MemAvailable_bytes
+      - node:node_memory_bytes_total:sum
+      - node:node_net_utilisation:sum_irate
+      - node_network_receive_bytes
+      - node_network_transmit_bytes
+      - node_disk_bytes_read
+      - node_disk_bytes_written
+      - node:node_disk_utilisation:avg_irate
+      - kube_pod_status_ready
+      - kube_pod_status_phase
+      - node_filesystem_size
+      - node_filesystem_avail
+      - kube_pod_container_status_restarts_total
+      - openshift_clusterresourcequota_usage
+      - openshift_clusterresourcequota_labels
+      - namespace_pod_name_container_name:container_cpu_usage_seconds_total:sum_rate
+      - kube_namespace_labels
+      - container_memory_rss
+      - container_network_receive_bytes_total
+      - container_network_transmit_bytes_total
+      - container_network_receive_packets_total
+      - container_network_transmit_packets_total
+      - container_network_receive_packets_dropped_total
+      - container_network_transmit_packets_dropped_total
+      #- kube_pod_info
+      - container_cpu_usage_seconds_total
+      #- kube_node_status_allocatable_cpu_cores
+      ### metrics for ocp 311 end
```

Comment:
Do the above metrics exist in OCP 4.x? in other word, does it have any impacts on performance?

Reply:
Most of these metrics also exist in ocp4.x.
And these starting with `kube_pod` and `container` are heavy metrics.

Reply:
Can we have a separated metrics_allowlist.yaml for OCP 3. for example: `ocp3_metrics_allowlist.yaml`. the mco-operator knows the managed cluster is OCP 3 or OCP 4 or *KS. so if the managed cluster is OCP 3, then pass ocp3 specific metrics allowlist, because we have many separated dashboards for OCP 3. there is no co-existing case so far. Does it make sense?

Reply:
sounds good to me @clyang82

Reply:
done.